modules/con_notifications-configuration.adoc

```diff
@@ -0,0 +1,100 @@
+:_mod-docs-content-type: CONCEPT
+[id="notifications-configuration_{context}"]
+= NotificationsConfiguration
+
+The `NotificationsConfiguration` is a Kubernetes custom resource (CR) that manages notifications in a Kubernetes cluster. In {gitops-title}, the `NotificationsConfiguration` (CR) allows users to add Templates, Triggers, Services, and Subscriptions resources to an Argo CD Notifications ConfigMap.
+
+When you create a cluster in {gitops-title} with notifications enabled, a `NotificationsConfiguration` (CR) is created by default with the name `default-notifications-configuration`.
+
+Any change made in the existing configuration of the `NotificationsConfiguration` is replicated in Argo CD Notifications ConfigMap. For example, if the user adds some trigger configuration in the `NotificationsConfiguration` resource, this configuration is read, processed, and updated in the Argo CD Notifications ConfigMap.
+
+[NOTE]
```

Comment:
```suggestion
[IMPORTANT]
```
Because in the admonition we mention words such as "must" and we also mention support, this should be important admonition: https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
Fixed.